CHANGELOG.md

```diff
@@ -6,7 +6,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
-
+- OSSE project: trajectory sampler (regrid to IODA file locations), capable of ingesting multiple files
```

Comment:
This comment needs to be categorized into one of the added/fixed/changed categories.

Reply:
I noticed after this comment that your description in the PR itself is quite nice.  I suggest pasting that description under `### Added`.